### ISSUE TITLE
Include an example on how to use tesseroid layers in the User Guide

### ISSUE THREAD

**Issue by santisoler** [fatiando/harmonica#352]
After #316 was merged we now can define tesseroid layers. The docs also include a gallery example that makes use of the new feature. Would be nice to have a section in the User Guide, similar to the one we have for the prism layer: https://www.fatiando.org/harmonica/latest/user_guide/forward_modelling/prism.html#prism-layer
